app/views/teaching_events/show.html.erb

```diff
@@ -0,0 +1,143 @@
+<div class="teaching-event">
+
+  <header class="yellow">
+
+      <div class="breadcrumbs-and-heading">
+        <%= render Header::BreadcrumbComponent.new %>
+        <h1><%= @event.name %></h1>
+      </div>
+
+      <div class="event-details">
+
+        <div class="date-and-time">
+          📅
+          <%= @event.start_at.to_formatted_s(:event) %>
+          at
+          <%= @event.start_at.to_formatted_s(:time) %> - <%= @event.end_at.to_formatted_s(:time) %>
+        </div>
+
+        <div class="location">
+          <% if @event.building.present? %>
+            <%= @event.building.venue %>
+            <%= @event.building.address_postcode %>
+          <% end %>
+
+          <% if @event.is_online %>
+            <div>
+              Online
+            </div>
+          <% end %>
+
+          <% if @event.is_virtual %>
+            <div>
+              Virtual
+            </div>
+          <% end %>
+        </div>
+
+        <div class="register">
+          <%= link_to event_steps_path(@event.readable_id), class: %w[button button--black-chevron] do %>
+            Register your attendence
+          <% end %>
+        </div>
+
+      </div>
+
+      <div class="image">
+        <%= image_pack_tag("media/images/content/hero-images/0017.jpg", alt: "A photograph of the venue", class: "event-info__venue_img") %>
+      </div>
+
+      <div class="triangle yellow">
+        <svg height="100%" width="100%" viewbox="0 0 100 100" preserveAspectRatio="none">
+          <polygon points="0,100 0,0 100,0" />
+        </svg>
+      </div>
+    </section>
+  </header>
+
+  <section class="event-info">
+    <h2>What to expect from a <%= event_type_name(@event.type_id) %> event</h2>
+
+    <div class="event-summary">
+      <div class="about-ttt-events">
+        Watch presentations, ask questions, speak to advisers one-to-one.
+      </div>
+
+      <figure>
+        <blockquote>
+          Absolutely loved this day. Very informative, inspiring and I'm
+          excited to change career. Thank you.
+        </blockquote>
+        <figcaption>Sarah, trainee teacher, London</figcaption>
+      </figure>
+    </div>
+
+    <%# <ul> %>
+    <%#   <li><%= @event.start_at %1></li> %>
+    <%#   <li><%= @event.readable_id %1></li> %>
+    <%#   <li><%= @event.name %1></li> %>
+    <%#   <li><%= @event.summary %1></li> %>
+    <%#   <li>online: <%= @event.is_online %1></li> %>
+    <%#   <li>virtual: <%= @event.is_virtual %1></li> %>
+    <%#   <li><%= @event.building&.venue %1></li> %>
+    <%#   <li><%= @event.building&.address_postcode %1></li> %>
+    <%# </ul> %>
+  </section>
+
+  <section class="event-attributes">
```

Comment:
I'm not convinced by this section - I think it is a lot of repeated information that they already know from earlier up the page; takes up a lot of real estate!